.gitignore

```diff
@@ -4,4 +4,6 @@ del*.au3
 Config/config.st
 theme.def
 2theme.def
-Backup/*
\ No newline at end of file
+Backup/*
```

Comment:
why these .gitignore changes?

Reply:
That's by mistake. I believe I did an additional commit to cancel it before submitting this pull request.